vars/slingOsgiBundleBuild.groovy

```diff
@@ -178,24 +178,27 @@ def defineStage(def globalConfig, def jobConfig, def jdkVersion, boolean isRefer
     def branchConfig = jobConfig?.branches?."$env.BRANCH_NAME" ?: [:]
 
     return {
-        wrapInNode(branchConfig.nodeLabel ?: globalConfig.mainNodeLabel, {
-            timeout(time: 30, unit: 'MINUTES') {
-                stage("Maven Build (Java ${jdkVersion}, ${goal})") {
-                    echo "Running on node ${env.NODE_NAME}"
-                    invocation.call()
+        node(branchConfig.nodeLabel ?: globalConfig.mainNodeLabel) {
+            dir(jenkinsJdkLabel) { // isolate parallel builds on same node
+                timeout(time: 30, unit: 'MINUTES') {
+                    checkout scm
+                    stage("Maven Build (Java ${jdkVersion}, ${goal})") {
+                        echo "Running on node ${env.NODE_NAME}"
+                        invocation.call()
```

Comment:
@rombert Should we also use isolated Maven repositories (https://www.jenkins.io/doc/pipeline/steps/pipeline-maven/#withmaven-provide-maven-environment, `mavenLocalRepo`?). That will lead to increased build times again, but not sure if Maven 3.8.6 really supports parallel access of repo yet (https://maven.apache.org/resolver/maven-resolver-named-locks/index.html)

Reply:
Isolated Maven repo in https://github.com/apache/sling-tooling-jenkins/pull/10/commits/ea1681b060a520def5411b42522d343856e1101b